x-pack/plugins/ml/public/application/jobs/new_job/pages/new_job/page.tsx

```diff
@@ -125,6 +125,7 @@ export const Page: FC<PageProps> = ({ existingJobsAndGroups, jobType }) => {
 
     if (jobCreator.type === JOB_TYPE.SINGLE_METRIC) {
       jobCreator.modelPlot = true;
+      jobCreator.modelChangeAnnotations = true;
```

Comment:
do we definitely want to enabling this for all single metric jobs?

Reply:
I think just enabling modelPlot (and omitting modelChangeAnnotations) will actually enable modelChangeAnnotations as well by default on elastic's side so I thought it would be clearer to state modelChangeAnnotations as true on the UI side for consistency. 

As for why we wanted both on for SM jobs, the discussion was that we expect annotations to be fairly high volume for high cardinality jobs, but nowhere near as high volume as model plot. So if we can tolerate the volume of model plot, we would probably also want annotations.

Reply:
>I think just enabling modelPlot (and omitting modelChangeAnnotations) will actually enable modelChangeAnnotations as well by default on elastic's side
This is no longer possible with the changes you added in 405c2a78b2bf5993432b6974c64519bfbac48f12. So if we do want to only enable `modelPlot` this line could be omitted.

I was asking more from a product design point of view, whether this had been discussed and it sounds like it has.